config/config.go

```diff
@@ -426,6 +434,35 @@ type Checkpoint struct {
 	Hash   types.Hash `toml:"hash" comment:"checkpoint block hash."`
 }
 
+type Erc20RewardSignerConfig struct {
+	Enable      bool           `toml:"enable" comment:"enable the ERC20 reward signer service"`
+	Targets     []string       `json:"targets" comment:"target reward ext alias for the ERC20 reward"`
+	PrivateKeys []string       `json:"private_keys" comment:"private key for the ERC20 reward target"`
+	SyncEvery   types.Duration `json:"sync_every" comment:"sync interval; a recommend value is same as the block time"`
```

Comment:
Since rn the block time is determined by two consensus parameters, not easy for the signerSvc to generate a good value for this configuration. So I'll leave to the user.